pkg/testutil/testutil_linux.go

```diff
@@ -33,6 +33,10 @@ var (
 	MariaDBImage                = mirrorOf("mariadb:10.5")
 	DockerAuthImage             = mirrorOf("cesanta/docker_auth:1.7")
 
+	NonDistBlobImage = "ghcr.io/cpuguy83/non-dist-blob:latest"
```

Comment:
Where is the source of this?

Reply:
https://gist.github.com/cpuguy83/fcf3041e5d8fb1bb5c340915aabeebe0

Reply:
Please explain the source in the code comments.

Reply:
Done